index.html

```diff
@@ -2512,22 +2507,20 @@ <h3>WoT Scripting API</h3>
                 time-critical logic from the cloud down to a gateway or
                 edge node.
             </p>
-            <p> The non-normative <a>WoT Scripting API</a> document defines
-                the structure and algorithms of the programming
-                interface that allows scripts to discover, process
-                (retrieve) and produce (define) <a>Thing Descriptions</a>. The
-                implementation of the <a>WoT Scripting API</a> instantiates local
+            <p> The non-normative <a>WoT Scripting API</a> document [[?wot-scripting-api]
+                defines the structure and algorithms of the programming
+                interface that allows scripts to discover, fetch and process,
+                and produce (i.e., define and publish) <a>WoT Thing Descriptions</a>. The
+                runtime system of the <a>WoT Scripting API</a> instantiates local
                 objects that act as an interface to other <a>Things</a> and
-                represents their <a>Interaction Affordances</a> (<a>Properties</a>,
+                their <a>Interaction Affordances</a> (<a>Properties</a>,
                 <a>Actions</a>, and <a>Events</a>). It also allows scripts to expose
-                <a>Things</a>, i.e. to specify and implement <a>Interaction
+                <a>Things</a>, that is, to define and implement <a>Interaction
                 Affordances</a> and publish a <a>Thing Description</a>.
             </p>
             <p> The standardized <a>WoT Scripting API</a> provides a
```

Comment:
this paragraph could be removed, as it does not add much here

Reply:
Removing.